### PR TITLE
[All Hosts] (manifest) removing out-of-date schemas and pointing links to the official copies

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
Links to the schemas have been pointed to the official schemas. These links won't work on the review site because that repo doesn't have a branch named removing-schemas.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f2b76ad](https://github.com/OfficeDev/office-js-docs-pr/commits/f2b76ad5d11468eb3fc6f3b82f25a5ba4d93aef1)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[_dependentPackages/CommonPlugins/tools/JoinTOC.ps1](https://github.com/OfficeDev/office-js-docs-pr/blob/removing-schemas/_dependentPackages/CommonPlugins/tools/JoinTOC.ps1) | :x:Error | | [Details](#user-content-402d9d56-30997783)
[.openpublishing.publish.config.json](https://github.com/OfficeDev/office-js-docs-pr/blob/removing-schemas/.openpublishing.publish.config.json) | :x:Error | | [Details](#user-content-d902ab53-30997783)
[]() | :x:Error | | [Details](#user-content-30997783)

<a id="402d9d56-30997783"></a>
### [_dependentPackages/CommonPlugins/tools/JoinTOC.ps1](https://github.com/OfficeDev/office-js-docs-pr/blob/removing-schemas/_dependentPackages/CommonPlugins/tools/JoinTOC.ps1)
 - **Line 82**: **[Error]** `````Error happened when executing plug-in script _dependentPackages/CommonPlugins/tools/JoinTOC.ps1 at step docset_prebuild. Please check your .openpublishing.publish.config.json?on the configuration of the plug-ins or ask for help?in Docs Support teams channel. Exit code: -1073741502`````
 - **Line 217**: **[Error]** `````Error happened when executing plug-in script _dependentPackages/CommonPlugins/tools/JoinTOC.ps1 at step docset_prebuild. Please check your .openpublishing.publish.config.json?on the configuration of the plug-ins or ask for help?in Docs Support teams channel. Please also include the build report and the following error code for our team to troubleshoot: 1`````

<a id="d902ab53-30997783"></a>
### [.openpublishing.publish.config.json](https://github.com/OfficeDev/office-js-docs-pr/blob/removing-schemas/.openpublishing.publish.config.json)
 - **Line 80**: **[Error]** `````Failed to execute customized script at step docset_prebuild in docset office-js-docs-pr. Please check your .openpublishing.publish.config.json?on the configuration of the plug-ins for docset office-js-docs-pr step docset_prebuild or ask for help?in Docs Support teams channel. Please also include the build report and the following error code for our team to troubleshoot: 1`````

<a id="30997783"></a>
###
 - **Line 138**: **[Error]** `````W:\wupm-s\.optemp\packages\opbuild.scripts.1.84.4\tools\opbuild\docset.prebuild.ps1 failed to execute`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202002131853181192-1594%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=XwBJdgf%2BuFIIxk%2BuTnbso2pnwHAOaZtb8YA3mwJfrgg%3D&st=2020-02-13T18%3A49%3A16Z&se=2020-03-15T18%3A54%3A16Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[706263e](https://github.com/OfficeDev/office-js-docs-pr/commits/706263ef3a977dde6fc3b7b88bdac852cfe6d32f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/troubleshoot-manifest.md](https://github.com/OfficeDev/office-js-docs-pr/blob/removing-schemas/docs/testing/troubleshoot-manifest.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C14%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202002141726411304-1594%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=DY3XyDm2MMdv2KtRVt1K7kY8L%2Bwo5MrNU2e%2FgVHU248%3D&st=2020-02-14T17%3A23%3A29Z&se=2020-03-16T17%3A28%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[951eb64](https://github.com/OfficeDev/office-js-docs-pr/commits/951eb6450fac553ea9418c6b3c447a49a057c9e6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/add-in-manifests.md](https://github.com/OfficeDev/office-js-docs-pr/blob/removing-schemas/docs/develop/add-in-manifests.md) | :white_check_mark:Succeeded | |
[docs/develop/manifest-element-ordering.md](https://github.com/OfficeDev/office-js-docs-pr/blob/removing-schemas/docs/develop/manifest-element-ordering.md) | :white_check_mark:Succeeded | |
[docs/overview/schemas/content/OfficeAppBasicTypesV1_0.xsd](https://github.com/OfficeDev/office-js-docs-pr/blob/removing-schemas/docs/overview/schemas/content/OfficeAppBasicTypesV1_0.xsd) | :white_check_mark:Succeeded | |
[docs/overview/schemas/content/OfficeAppManifestV1_1.xsd](https://github.com/OfficeDev/office-js-docs-pr/blob/removing-schemas/docs/overview/schemas/content/OfficeAppManifestV1_1.xsd) | :white_check_mark:Succeeded | |
[docs/overview/schemas/mail/MailAppVersionOverridesV1_0.xsd](https://github.com/OfficeDev/office-js-docs-pr/blob/removing-schemas/docs/overview/schemas/mail/MailAppVersionOverridesV1_0.xsd) | :white_check_mark:Succeeded | |
[docs/overview/schemas/mail/MailAppVersionOverridesV1_1.xsd](https://github.com/OfficeDev/office-js-docs-pr/blob/removing-schemas/docs/overview/schemas/mail/MailAppVersionOverridesV1_1.xsd) | :white_check_mark:Succeeded | |
[docs/overview/schemas/mail/OfficeAppBasicTypesV1_0.xsd](https://github.com/OfficeDev/office-js-docs-pr/blob/removing-schemas/docs/overview/schemas/mail/OfficeAppBasicTypesV1_0.xsd) | :white_check_mark:Succeeded | |
[docs/overview/schemas/mail/OfficeAppManifestV1_1.xsd](https://github.com/OfficeDev/office-js-docs-pr/blob/removing-schemas/docs/overview/schemas/mail/OfficeAppManifestV1_1.xsd) | :white_check_mark:Succeeded | |
[docs/overview/schemas/taskpane/MailAppVersionOverridesV1_0.xsd](https://github.com/OfficeDev/office-js-docs-pr/blob/removing-schemas/docs/overview/schemas/taskpane/MailAppVersionOverridesV1_0.xsd) | :white_check_mark:Succeeded | |
[docs/overview/schemas/taskpane/OfficeAppBasicTypesV1_0.xsd](https://github.com/OfficeDev/office-js-docs-pr/blob/removing-schemas/docs/overview/schemas/taskpane/OfficeAppBasicTypesV1_0.xsd) | :white_check_mark:Succeeded | |
[docs/overview/schemas/taskpane/OfficeAppManifestV1_1.xsd](https://github.com/OfficeDev/office-js-docs-pr/blob/removing-schemas/docs/overview/schemas/taskpane/OfficeAppManifestV1_1.xsd) | :white_check_mark:Succeeded | |
[docs/overview/schemas/taskpane/TaskPaneAppVersionOverridesV1_0.xsd](https://github.com/OfficeDev/office-js-docs-pr/blob/removing-schemas/docs/overview/schemas/taskpane/TaskPaneAppVersionOverridesV1_0.xsd) | :white_check_mark:Succeeded | |
[docs/testing/troubleshoot-manifest.md](https://github.com/OfficeDev/office-js-docs-pr/blob/removing-schemas/docs/testing/troubleshoot-manifest.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C14%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202002141754233831-1594%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=gFmetca8aqvTgR5AZNzq2O%2BCre1XL7TRE9fnBjUHQx4%3D&st=2020-02-14T17%3A51%3A16Z&se=2020-03-16T17%3A56%3A16Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/manifest-element-ordering.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ For example, in the `<OfficeApp>` element, the `<Id>`, `<Version>`, `<ProviderNa
 The following sections show the manifest elements in the order in which they must appear. There are differences depending on whether the `type` attribute of the `<OfficeApp>` element is `TaskPaneApp`, `ContentApp`, or `MailApp`. To keep these sections from becoming too unwieldy, the highly complex `<VersionOverrides>` element is broken out into separate sections.
 
 > [!Note]
-> Not all of the elements shown are mandatory. If the `minOccurs` value for a element is **0** in the [schema](https://github.com/OfficeDev/office-js-docs-pr/tree/master/docs/overview/schemas), the element is optional.
+> Not all of the elements shown are mandatory. If the `minOccurs` value for a element is **0** in the [schema](/openspecs/office_file_formats/ms-owemxml/4e112d0a-c8ab-46a6-8a6c-2a1c1d1299e3), the element is optional.
```
Comment (kbrandl) [inline docs/develop/manifest-element-ordering.md]: All of the other links go to the schemas landing page, but this link goes directly to a schema page. I assume this is intentional?

Comment (Rick-Kirkham) [reply docs/develop/manifest-element-ordering.md]: Yes. This is the one case where it is feasible to be more specific in the link target.

**Pull request merged**